src/css/math.less

```diff
@@ -298,6 +298,21 @@
     }
   }
 
+  .mq-limit {
+    text-align: center;
+
+    span  {
+      display: block;
```

Comment:
ok I pushed it, but I don't really get your last example - it seemed to work fine